research/conditional/train/cc_train.py

```diff
@@ -159,6 +159,12 @@ def main(
         else:
             raise ValueError(f"Unknown module name: {module_name}")
 
+    if args.parallel_blocks:
+        modules = block_modules.values()
+        block_modules = {
+            "parallel": lambda: Parallel(*[module() for module in modules])
```

Comment:
I think this makes both blocks share the same layernorm. Maybe it's good, maybe it isn't; probably doesn't matter too much. But it's at least worth noting.

Reply:
Intuitively, I would like both of those blocks to have separate layernorms, but I'm not sure.